content/zh/docs/community/localization.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "本地化 Helm 文档"
+description: "本指南说明了如何本地化 Helm 文档。"
+aliases: ["/docs/localization/"]
```

Comment:
Please remove this alias; it will not do what you think it will do. (It will redirect the English-language localization page to this new page, which is not what we want.)